src/commands/apextests/list.ts

```diff
@@ -56,25 +71,31 @@ export default class ApextestsList extends SfCommand<ApextestsListResult> {
       throw new Error('Invalid directory.');
     }
 
-    const files = readDir.filter((file) => file.endsWith('.cls') || file.endsWith('.trigger'));
-    const testMethodsNames: string[] = [];
+    const localFiles = readDir.filter((file) => file.endsWith('.cls') || file.endsWith('.trigger'));
+
+    const handler = (fileName: string): void => {
+      const path = `${directory}/${fileName}`;
+      const data = fs.readFileSync(path, 'utf-8');
+      const testMethods = data.match(TEST_NAME_REGEX);
 
-    // read each file and check for the test methods at the top
-    files.map((file) => {
-      const data = fs.readFileSync(`${directory}/${file}`, 'utf8');
+      testMethodsNames.push(...(testMethods ? ApextestsList.parseTestsNames(testMethods) : []));
```

Comment:
**suggestion:** with the new version of parseTestsNames it can be written simpler

```suggestion
      testMethodsNames.push(...ApextestsList.parseTestsNames(testMethods));
```